clib/ancient.mli

```diff
@@ -0,0 +1,19 @@
+(************************************************************************)
+(*         *   The Coq Proof Assistant / The Coq Development Team       *)
+(*  v      *   INRIA, CNRS and contributors - Copyright 1999-2019       *)
+(* <O___,, *       (see CREDITS file for the list of authors)           *)
+(*   \VV/  **************************************************************)
+(*    //   *    This file is distributed under the terms of the         *)
+(*         *     GNU Lesser General Public License Version 2.1          *)
+(*         *     (see LICENSE file for the text of the license)         *)
+(************************************************************************)
+
+type 'a t
+
+val make : 'a -> 'a t
+
+val get : 'a t -> 'a
+
+val delete : 'a t -> unit
```

Comment:
Why do we need this?

Reply:
Not to clutter the disk with stale files.

Reply:
But the GC would call the finalizer anyway no? Are you afraid we hit the maximum number of open files per process?

Reply:
The GC doesn't always run when the program exits, so that in a typical run you get random remaining files on your hard drive.